dynawo/sources/Modeler/DataInterface/PowSyblIIDM/test/TestLineInterface.cpp

```diff
@@ -173,6 +173,13 @@ TEST(DataInterfaceTest, Line) {
   li.addCurrentLimitInterface2(curLimItf2);
   ASSERT_EQ(li.getCurrentLimitInterfaces1().size(), 1);
   ASSERT_EQ(li.getCurrentLimitInterfaces2().size(), 1);
+  std::string season = "UNDEFINED";
```

Comment:
why creating a temporary variable?

Reply:
No reason, I just find it easier to understand what the UNDEFINED stands for if we have a variable saying it is "season"